ui/vic-ui-service/.classpath

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I initially dropped all Eclipse config files in the repo but then had a problem with syncing with Josh Kenney's forked repo. We created the same 'ui' branch and pulled each other's work. We'll now work individually on each other's branch so that we don't need to check in such files. I can remove these config files from the `ui-onsite-beta` branch, as this has branched off from the original `ui` branch.